### PR TITLE
chore: dependency updates, eslint v8, node 18

### PR DESCRIPTION
**Pull request by C0rydoras**

### CODE REVIEW

File: frontend/.eslintrc.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
 "use strict";
 
 module.exports = {
-  extends: "@adfinis-sygroup/eslint-config/ember-app",
-
+  settings: {
+    "import/internal-regex": "^customer-center/",
+  },
+  extends: ["@adfinis/eslint-config/ember-addon"],
```
Comment (derrabauke) [inline frontend/.eslintrc.js:7]:
```suggestion
  extends: ["@adfinis/eslint-config/ember-app"],
```

It's an `app`, not a `addon`.

File: frontend/.eslintrc.js [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
   rules: {
-    "no-unused-vars": "off",
-    "prefer-rest-params": "off",
-    "ember/use-brace-expansion": "off",
-    "ember/no-get": "off",
-    "ember/no-mixins": "warn",
```
Comment (derrabauke) [inline frontend/.eslintrc.js:11]: I think we still need them.

**Review by derrabauke — CHANGES_REQUESTED** [review]
There are some warning which may give you a hint what `deps` need an update as well:
```bash
warning " > @adfinis/eslint-config@2.0.0" has incorrect peer dependency "eslint-config-prettier@^8.8.0".
warning " > @adfinis/eslint-config@2.0.0" has incorrect peer dependency "eslint-plugin-ember@^11.5.2".
warning " > @adfinis/eslint-config@2.0.0" has incorrect peer dependency "eslint-plugin-import@^2.27.5".
warning " > @adfinis/eslint-config@2.0.0" has incorrect peer dependency "prettier@^2.8.8".
```

File: frontend/.eslintrc.js (hunk 2 above)
Comment (C0rydoras) [reply frontend/.eslintrc.js]: Re-added as of https://github.com/adfinis/customer-center/pull/412/commits/179426966a197d6f2126c01d5767b25b74542d4f

File: frontend/.eslintrc.js (hunk 1 above)
Comment (C0rydoras) [reply frontend/.eslintrc.js]: Done in https://github.com/adfinis/customer-center/pull/412/commits/179426966a197d6f2126c01d5767b25b74542d4f

**Comment by derrabauke** [comment]
Just add the `hbs` errors as expections to [the config](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/configuration.md)

File: frontend/.template-lintrc.js [hunk 3]
```diff
@@ -1,11 +1,98 @@
...
+    "require-lang-attribute": "warn",
+    "require-mandatory-role-attributes": "warn",
+    "require-media-caption": "warn",
+    "require-presentational-children": "warn",
+    "require-valid-alt-text": "warn",
+    "require-valid-named-block-naming-format": "warn",
+    "simple-unless": "warn",
+    "splat-attributes-only": "warn",
+    "style-concatenation": "warn",
+    "table-groups": "warn",
```
Comment (derrabauke) [inline frontend/.template-lintrc.js:96]: This is all needed? :fearful:

Comment (C0rydoras) [reply frontend/.template-lintrc.js:96]: AFAIK for everything to be a warning yes, not sure tho

**Review by derrabauke — APPROVED** [review]

**Comment by derrabauke** [comment]
Don't forget to squash to keep the history clean ;)

**Pull request merged**